### ISSUE TITLE
darker/import_sorting.py throws isort_code = getattr(isort, "code") AttributeError: 'NoneType' object has no attribute 'code'

### ISSUE THREAD

**Issue by irynahryshanovich** [akaihola/darker#145]
Hello, 
I use darker rev: 1.2.3 and isort rev: 5.8.0 in pre-commit configurations. Every time when I run command:
 pre-commit run --all-files

I get this error:

```
(venv) 4c-mac-irynahryshanovich:gauge-qa-automation irynahryshanovich$ pre-commit run --all-files
[INFO] Initializing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Initializing environment for https://gitlab.com/pycqa/flake8.
[INFO] Initializing environment for https://github.com/pycqa/isort.
[INFO] Initializing environment for https://github.com/akaihola/darker.
[INFO] Installing environment for https://github.com/pre-commit/pre-commit-hooks.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://gitlab.com/pycqa/flake8.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/pycqa/isort.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
[INFO] Installing environment for https://github.com/akaihola/darker.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check Yaml...............................................................Passed
Check for added large files..............................................Passed
Fix requirements.txt.....................................................Passed
flake8...................................................................Passed
isort....................................................................Passed
darker...................................................................Failed
- hook id: darker
- exit code: 1

Traceback (most recent call last):
  File "/Users/irynahryshanovich/.cache/pre-commit/reponafqqqvh/py_env-python3/bin/darker", line 5, in <module>
    from darker.__main__ import main
  File "/Users/irynahryshanovich/.cache/pre-commit/reponafqqqvh/py_env-python3/lib/python3.7/site-packages/darker/__main__.py", line 16, in <module>
    from darker.import_sorting import apply_isort, isort
  File "/Users/irynahryshanovich/.cache/pre-commit/reponafqqqvh/py_env-python3/lib/python3.7/site-packages/darker/import_sorting.py", line 22, in <module>
    isort_code = getattr(isort, "code")
AttributeError: 'NoneType' object has no attribute 'code'

```
I saw some changes were made in import_sorting.py file recently. Will they fix my problem and if so how soon they will be released?

Thank you!

**Comment by dsmanl** [comment]
Dear all,

Same issue here. Is there any update on this issue?
Thanks a lot for your effort!

Best regards and stay healthy!

**Comment by akaihola** [comment]
Thanks @irynahryshanovich and @dsmanl for your report! This was fixed in #132 and will be included in version 1.2.4. See #144 for remaining merge requests for that release – you can help by reviewing them.
